File: Test Code_GPT/test21.cpp
#include <iostream>

class BankAccount {
private:
    double balance;
    int totalTransactions;

public:
    // Constructor
    BankAccount(double initialBalance) : balance(initialBalance), totalTransactions(0) {}

    // Deposit funds
    void deposit(double amount) {
        balance += amount;
        totalTransactions++;
    }

    // Withdraw funds
    void withdraw(double amount) {
        if (amount <= balance) {
            balance -= amount;
            totalTransactions++;
        } else {
            std::cout << "Insufficient funds for withdrawal." << std::endl;
        }
    }

    // Get current balance
    double getBalance() const {
        return balance;
    }

    // Get total number of transactions
    int getTotalTransactions() const {
        return totalTransactions;
    }
};

int main() {
    // Example usage
    BankAccount account(1000.0);
    
    account.deposit(500.0);
    account.withdraw(200.0);
    
    std::cout << "Current Balance: " << account.getBalance() << std::endl;
    std::cout << "Total Transactions: " << account.getTotalTransactions() << std::endl;

    return 0;
}

File: Test Code_codellama/Test Python/test21.py
class BankAccount:
   def __init__(self, account_number, initial_balance):
       self.account_number = account_number
       self.balance = initial_balance
       self.total_accounts = 0

   def deposit(self, amount):
       self.balance += amount

   def withdraw(self, amount):
       self.balance -= amount

   def get_balance(self):
       return self.balance

   def total_accounts(self):
       return self.total_accounts